source/codegen/validate-linux-rt.py

```diff
@@ -36,9 +37,12 @@ def _get_non_rt_driver_changes(driver_name: str, changed_files: Set[str]) -> Set
 
 def _need_linux_rt_feed_update(metadata_dir: str, changed_files: Set[str]) -> bool:
     codegen_changes = _get_codegen_changes(changed_files)
+    _print_changed_files("Codegen changes not affecting RT:", codegen_changes)
```

Comment:
[Note] I went back and added these print statements to give more information in the GitHub action output (look under the section "Update Linux RT's grpc-device dependency to v1.5.2?" in https://github.com/reckenro/grpc-device/runs/7024760828?check_suite_focus=true.